com.viber.Viber.json

```diff
@@ -0,0 +1,66 @@
+{
+  "app-id": "com.viber.Viber",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "viber",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--filesystem=home",
+    /* needed for video chat */
+    "--device=all",
+    "--talk-name=org.freedesktop.Notifications",
```

Comment:
Does this have a tray icon? If so you'll need the appindicator bits (see discord flatpak).

Reply:
Yes, this does. Will add that.